lib/double_entry/reporting/aggregate_array.rb

```diff
@@ -50,8 +53,9 @@ def retrieve_aggregates
         scope = LineAggregate.
                 where(:function => function).
                 where(:range_type => 'normal').
-                where(:account => account.to_s).
-                where(:code => code.to_s).
+                where(:account => account.try(:to_s)).
+                where(:code => code.try(:to_s)).
+                where(:partner_account => partner_account.try(:to_s)).
```

Comment:
Sorry I was mid-writing this comment and never submit it in response to @macosgrove 's comment on the try's

So, there's weirdness that @salamagd  and I discovered:
```
irb(main):001:0> nil.to_s
=> ""
irb(main):002:0> nil.try(:to_s)
=> nil
```

We're storing NULL in the DB for code/partner_account if they aren't part of the aggregate, but then here we were looking for "" which will result in